{{cookiecutter.project_slug}}/README.md

```diff
@@ -1,5 +1,43 @@
 # {{ cookiecutter.project_name }}
 
+# Description
+
 {{ cookiecutter.project_short_description }}
 
 Created by {{ cookiecutter.full_name }} ({{ cookiecutter.email }})
+
+# Development Environment
+
+This project is designed to run on [Neuro Platform](https://neu.ro), so you can jump into problem-solving right away.
+
+## Directory Structure
+
+| Mount Point                                  | Description           | Storage URI                                                                  |
+|:-------------------------------------------- |:--------------------- |:---------------------------------------------------------------------------- |
+|`/project/data/`                              | Data                  | `storage:{{ cookiecutter.project_slug }}/data/`                              |
+|`/project/{{ cookiecutter.code_directory }}/` | Python modules        | `storage:{{ cookiecutter.project_slug }}/{{ cookiecutter.code_directory }}/` |
+|`/project/notebooks/`                         | Jupyter notebooks     | `storage:{{ cookiecutter.project_slug }}/notebooks/`                         |
+|`/project/results/`                           | Logs and results      | `storage:{{ cookiecutter.project_slug }}/results/`                           |
+
+## Development
+
+Follow the instructions below to setup the environment and start Jupyter development session.
```

Comment:
Minor:

> "Follow the instructions below to setup"

hard to read. Maybe better:
- Follow the instructions below *in order to* setup ...
- Follow the instructions below *for setting up the project* ...